make-based/app-elfloader/files/.config

```diff
@@ -0,0 +1,358 @@
+#
+CONFIG_ARCH_X86_64=y
+# CONFIG_ARCH_ARM_64 is not set
+# CONFIG_ARCH_ARM_32 is not set
+# CONFIG_MARCH_X86_64_NATIVE is not set
+CONFIG_MARCH_X86_64_GENERIC=y
+# CONFIG_MARCH_X86_64_NOCONA is not set
+# CONFIG_MARCH_X86_64_CORE2 is not set
+# CONFIG_MARCH_X86_64_COREI7 is not set
+# CONFIG_MARCH_X86_64_COREI7AVX is not set
+# CONFIG_MARCH_X86_64_COREI7AVXI is not set
+# CONFIG_MARCH_X86_64_ATOM is not set
+# CONFIG_MARCH_X86_64_K8 is not set
+# CONFIG_MARCH_X86_64_K8SSE3 is not set
+# CONFIG_MARCH_X86_64_AMDFAM10 is not set
+# CONFIG_MARCH_X86_64_BTVER1 is not set
+# CONFIG_MARCH_X86_64_BDVER1 is not set
+# CONFIG_MARCH_X86_64_BDVER2 is not set
+# CONFIG_MARCH_X86_64_BDVER3 is not set
+# CONFIG_MARCH_X86_64_BTVER2 is not set
+
+#
+# Processor Features
+#
+# CONFIG_X86_64_HAVE_RANDOM is not set
+# end of Processor Features
+
+CONFIG_STACK_SIZE_PAGE_ORDER=4
+# end of Architecture Selection
+
+#
+# Platform Configuration
+#
+CONFIG_PLAT_KVM=y
+
+#
+# Console Options
+#
+CONFIG_KVM_KERNEL_SERIAL_CONSOLE=y
+CONFIG_KVM_KERNEL_VGA_CONSOLE=y
+CONFIG_KVM_DEBUG_SERIAL_CONSOLE=y
+CONFIG_KVM_DEBUG_VGA_CONSOLE=y
+
+#
+# Serial console configuration
+#
+CONFIG_KVM_SERIAL_BAUD_115200=y
+# CONFIG_KVM_SERIAL_BAUD_57600 is not set
+# CONFIG_KVM_SERIAL_BAUD_38400 is not set
+# CONFIG_KVM_SERIAL_BAUD_19200 is not set
+# end of Serial console configuration
+# end of Console Options
+
+CONFIG_KVM_MAX_IRQ_HANDLER_ENTRIES=8
+CONFIG_KVM_PCI=y
+CONFIG_VIRTIO_BUS=y
+
+#
+# Virtio
+#
+CONFIG_VIRTIO_PCI=y
+CONFIG_VIRTIO_NET=y
+CONFIG_VIRTIO_9P=y
+# end of Virtio
+
+CONFIG_UKPLAT_ALLOW_GIC=y
+
+#
+# Platform Interface Options
+#
+# CONFIG_UKPLAT_MEMRNAME is not set
+CONFIG_UKPLAT_LCPU_MAXCOUNT=1
+CONFIG_PAGING=y
+# CONFIG_PAGING_5LEVEL is not set
+# CONFIG_PAGING_STATS is not set
+CONFIG_HAVE_PAGING=y
+CONFIG_HAVE_PAGING_DIRECTMAP=y
+# end of Platform Interface Options
+
+CONFIG_HZ=100
+# end of Platform Configuration
+
+#
+# Library Configuration
+#
+CONFIG_LIB9PFS=y
+# CONFIG_LIBDEVFS is not set
+# CONFIG_LIBFDT is not set
+CONFIG_LIBISRLIB=y
+CONFIG_LIBNOLIBC=y
+CONFIG_LIBNOLIBC_UKDEBUG_ASSERT=y
+CONFIG_LIBPOSIX_EVENT=y
+# CONFIG_LIBPOSIX_FUTEX is not set
+# CONFIG_LIBPOSIX_LIBDL is not set
+CONFIG_LIBPOSIX_MMAP=y
+# CONFIG_LIBPOSIX_MMAP_TEST is not set
+CONFIG_LIBPOSIX_PROCESS=y
+CONFIG_LIBPOSIX_PROCESS_PIDS=y
+CONFIG_LIBPOSIX_PROCESS_MAX_PID=32
+# CONFIG_LIBPOSIX_PROCESS_INIT_PIDS is not set
+CONFIG_LIBPOSIX_PROCESS_CLONE=y
+# CONFIG_LIBPOSIX_PROCESS_DEBUG is not set
+CONFIG_LIBPOSIX_SOCKET=y
+# CONFIG_LIBPOSIX_SOCKET_PRINT_ERRORS is not set
+CONFIG_LIBPOSIX_SYSINFO=y
+CONFIG_LIBPOSIX_TIME=y
+CONFIG_LIBPOSIX_USER=y
+# CONFIG_LIBRAMFS is not set
+CONFIG_LIBSYSCALL_SHIM=y
+# CONFIG_LIBSYSCALL_SHIM_LIBCSTUBS is not set
+CONFIG_LIBSYSCALL_SHIM_HANDLER=y
+CONFIG_LIBSYSCALL_SHIM_HANDLER_ULTLS=y
+
+#
+# Debugging
+#
+# CONFIG_LIBSYSCALL_SHIM_DEBUG_SYSCALLS is not set
+# CONFIG_LIBSYSCALL_SHIM_DEBUG_HANDLER is not set
+CONFIG_LIBSYSCALL_SHIM_STRACE=y
```

Comment:
Should we enable this in the non-debug `.config` file?
I think it gives a bit too much output, I would not use it in the non-debug configuration.

Reply:
We can create three configuration files: plain, strace, and full debug. It's very important to have this one for debugging. The debug one is too verbose.